### PR TITLE
Added python implementation to check UTF-8 encoding

### PR DESCRIPTION
**Pull request by rachitmanas**
## Assigned Issue Number?
#228 

## Language Opted?
Python3

## Number of files added or updated?
1

## Name of these files?
Algorithms => Bit-Manipulation => UTF-8 encoding checker =>UTF_8_encdng_chck.py

### Checklist

- [x] I was assigned for this otherwise my **PR will be rejected**.
- [x] I've read the contribution guidelines.
- [x] I've written code by myself. **No Plagiarism allowed**
- [x] I've filled up this entire template correctly.

- [x] I've referred the correct issue number.

:star2: this repository and support.

### CODE REVIEW

**Comment by krishrahul98** [comment]
@rachitmanas Please write your own implementation to check UTF-8 encoding. You are directly using Python function to check.

**Comment by krishrahul98** [comment]
@rachitmanas Please implement again by yourself and raise PR again.

File: Algorithms/Bit-Manipulation/UTF-8 encoding checker/UTF_8_encdng_chck.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+s=input("Enter the string to check the utf-8 validation for: ")
+valid_utf8 = True
+try:
+    s.encode().decode('utf-8')
```
Comment (krishrahul98) [inline Algorithms/Bit-Manipulation/UTF-8 encoding checker/UTF_8_encdng_chck.py:4]: Don't Use Direct python function.

**Review by krishrahul98 — CHANGES_REQUESTED** [review]
Please implement by yourself.

File: Algorithms/Bit-Manipulation/UTF-8 encoding checker/UTF_8_encdng_chck.py (hunk 1 above)
Comment (rachitmanas) [reply Algorithms/Bit-Manipulation/UTF-8 encoding checker/UTF_8_encdng_chck.py]: Please check. I have updated the code, removing any direct usage of inbuilt functions!

**Review by krishrahul98 — APPROVED** [review]

**Review by krishrahul98 — APPROVED** [review]
Great Work 👍🏼

**Pull request merged**